<ui version="4.0" >
 <class>numbereditClass</class>
 <widget class="QDialog" name="numbereditClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>数字编辑</string>
  </property>
  <widget class="QPushButton" name="backpb" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>511</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>数字属性</string>
   </property>
   <widget class="QLineEdit" name="le_targetvar" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_fudianshu" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>浮点数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_zhengshu" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>整数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>目标变量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_targetvar" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>70</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>20</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>整数位</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_zhengshuwei" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_xiaoshuwei" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>20</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>小数位</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_shuru" >
    <property name="geometry" >
     <rect>
      <x>350</x>
      <y>70</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>支持输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_fanweijiancha" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>范围检查</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_zuidazhi" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_zuixiaozhi" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>140</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>最大值</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_ziti" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>10</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>设置字体</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>数字颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_wenzicolor" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>边框背景颜色</string>
   </property>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>边框颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_biankuangyanse" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_juxingyanse" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>边框厚度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_biankuanghoudu" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>矩形颜色</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
